test/asset.ts

```diff
@@ -0,0 +1,203 @@
+import {assert} from 'chai'
+import {
+    Asset as AntelopeAsset,
+    APIClient,
+    FetchProvider,
+    Int64,
+    Name,
+    Serializer,
+} from '@wharfkit/antelope'
+import {Chains} from '@wharfkit/common'
+import {mockFetch} from '@wharfkit/mock-data'
+import {PlaceholderAuth} from '@wharfkit/signing-request'
+import type {Asset} from '$lib'
+import {
+    AtomicAssetsAPIClient,
+    AtomicAssetsContract,
+    AtomicAssetsKit,
+    AtomicUtility,
+    Types,
+} from '$lib'
+
+const client = new APIClient({
+    provider: new FetchProvider(Chains.WAX.url, {fetch: mockFetch}),
+})
+
+// Setup the API
+const atomicassets = new AtomicAssetsAPIClient(
+    new APIClient({
+        provider: new FetchProvider('https://wax.api.atomicassets.io/', {fetch: mockFetch}),
+    })
+)
+
+const utility = new AtomicUtility('https://wax.api.atomicassets.io/', Chains.WAX, {
+    client,
+    atomicClient: atomicassets,
+})
+
+const kitInst = new AtomicAssetsKit(utility)
+const accountName = 'test.gm'
+const assetId = 1099851897196
+
+suite('Asset', function () {
+    this.slow(200)
+    this.timeout(10 * 10000)
```

Comment:
I know you probably took this code from one of our packages, but 100 seconds timeout seems a bit excessive here 😄

Reply:
Fixed